merkle-patricia-tree/src/Data/Tree/MerklePatricia.hs

```diff
@@ -343,7 +347,40 @@ delete el newRoot proofs = do
 
 -- | Pretty-print a 'MerklePatriciaTree' in ascii, such that levels are aligned
 -- horizontally (one identation-level per level in the tree), and prefixes are
--- positioned such that they are visually easy to follow.
+-- positioned such that they are visually easy to follow. The outputs look like
+-- this:
+--
+-- @
+-- *--- Root hash
+-- |                      *---- Values at the leaves
+-- |                      |
+-- *-> #3801f:            |
+--     #ee41c: \\0         |
```

Comment:
If you paint it `0\` (or with another branch indicator) the bit's position would be aligned along all branches.

Reply:
It is better in the below example (of the walk backwards).. somehow you have two `\\` here

Reply:
Yes that's because this one rendered by haddock and I need to escape the backslashes..